keel-deliveryconfig-plugin/src/main/kotlin/com/netflix/spinnaker/keel/deliveryconfig/resource/DeliveryConfigHandler.kt

```diff
@@ -9,17 +9,22 @@ import com.netflix.spinnaker.keel.api.deliveryconfig.DeliveryConfig
 import com.netflix.spinnaker.keel.front50.Front50Service
 import com.netflix.spinnaker.keel.front50.model.Delivery
 import com.netflix.spinnaker.keel.plugin.ResourceHandler
+import com.netflix.spinnaker.keel.plugin.ResourceValidator
 import com.netflix.spinnaker.keel.retrofit.isNotFound
 import de.danielbechler.diff.node.DiffNode
 import de.huxhorn.sulky.ulid.ULID
 import kotlinx.coroutines.runBlocking
+import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import retrofit2.HttpException
 
 class DeliveryConfigHandler(
   private val front50Service: Front50Service,
-  override val objectMapper: ObjectMapper
+  override val objectMapper: ObjectMapper,
+  override val validators: List<ResourceValidator<DeliveryConfig>>
```

Comment:
Would it make sense to wire these into `ResourceActuator` rather than each individual handler?

Reply:
Discussed offline: going to move to the `ResourcePersister`

Reply:
@cfieber discussed offline with @robfletcher: Wiring into the `ResourcePersister` actually made things clunky because the naming filling in of metadata still needed to happen with all the knowledge that the handlers had (https://github.com/spinnaker/keel/blob/master/keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/ResourceHandler.kt#L48). Instead I've simplified the wiring and all the validation happens in the base `ResourceHandler` class.